# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: supply-chain/config.toml
# Commit: 9493029c76b9
# Full Hash: 9493029c76b93861ca10dbfb29d4178773144416
# Author: zaggy1024 <Zaggy1024@gmail.com>
# Date: 2023-01-28 21:11:06
# Regressor Bug: 1788119
# File Overlap Count: 1
# Overlapping Files: Cargo.lock
# Description:
#   Bug 1788119 - Part 4 - Update mp4parse-rust for AVIS support. r=kinetik,glandium,supply-chain-reviewers
#   
#   Depends on D156652
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D156653
# ==============================================================================

diff -r 7d5cc211e3f5 -r 9493029c76b9 supply-chain/config.toml
--- a/supply-chain/config.toml	Fri Jan 27 19:54:29 2023 +0000
+++ b/supply-chain/config.toml	Fri Jan 27 19:54:29 2023 +0000
@@ -671,10 +671,6 @@
 version = "1.0.2"
 criteria = "safe-to-run"
 
-[[exemptions.humantime]]
-version = "2.1.0"
-criteria = "safe-to-deploy"
-
 [[exemptions.hyper]]
 version = "0.14.19"
 criteria = "safe-to-run"